templates/kube-apiserver.yaml

```diff
@@ -29,6 +29,15 @@ spec:
         - --authorization-policy-file=/etc/auth/policies.jsonl
         - --authorization-rbac-super-user=kube-admin
         - --audit-log-path=/dev/stdout
+        - --audit-log-maxsize=50
+        - --audit-log-maxbackup=0
+      livenessProbe:
+        httpGet:
+          host: 127.0.0.1
+          port: 8080
+          path: /healthz
+        initialDelaySeconds: 15
+        timeoutSeconds: 15
```

Comment:
The periodSeconds and TimeoutSeconds could have a race. timeoutSeconds would be reached even before there is chance to check (since periodSeconds is 15 and could be run after the timeoutseconds)